### ISSUE TITLE
مشکل در subMonths

### ISSUE THREAD

**Issue by SadeghPM** [morilog/jalali#71]
درود.

طبق مثال readme :
```php
$date = (new Jalalian(1397, 1, 18))->subMonths(1)->toString() 
// output: 1396-12-18 00:00:00
```

اما اگر همین مثال رو اجرا کنید جوابی که میگیرید متفاوت هست.

```php
>>> (new  Morilog\Jalali\Jalalian(1397, 1, 18))->subMonths(1)->toString()
=> "1397-01-18 00:00:00"
```

**Comment by dena-a** [comment]
Hello @morilog,
This bug has been fixed on pull request #72 .
Please merge.
Thanks

**Comment by skhalili** [comment]
There is this bug and fixed by pull request #72

**Comment by morilog** [comment]
the pull request closed because of CI has been failed

**Comment by dena-a** [comment]
@skhalili The bug fixed on pull request #73. The issue has been solved.
@morilog Please close the issue.
